dashboard/app/models/script.rb

```diff
@@ -826,9 +839,10 @@ def has_lesson_pdf?
 
   def has_banner?
     # Temporarily remove Course A-F banner (wrong size) - Josh L.
```

Comment:
This [temporary change has been kickin' around for almost 2 years](https://github.com/code-dot-org/code-dot-org/pull/16621). @joshlory or maybe @ryansloan do you know if it's still needed? Should we add re-sizing banners for Course A-F to the LP board?

Reply:
Everything is temporary...
We should get banners on Mark's backlog and make sure they're correctly sized, then we can remove this temporary change. I think @fontie715 should drive, LMK if I can help!